main.js

```diff
@@ -104,22 +107,42 @@ async function main() {
     const $timeoutInput = $('#timeout-input');
```

Comment:
Base on the comments below, I image initialization should move to near the top of the function.

```
const $timeoutRange = $('#timeout-range');
const $timeoutInput = $('#timeout-input');
const $heatmapInput = $('#heatmap-input')
...
const alertAudioCookie = getCookies('alertAudio');
let alertAudio = getHowl(alertAudioCookie);
$soundInput.val(alertAudioCookie || 'built-in');
```

I would also probably define another function to refactor the audio declaration
```
function getHowl(name) {
  if (name === 'pop') {
    return new Howl({ src: [popUrl], html5: true });
  } else if (name == 'coronavirus') {
    return new Howl({ src: [coronavirusUrl], html5: true });
  } else {
    return null;
  }
}
```
I would also define another function to keep the cookie fresh when getting them, so that when people visit the page, the cookies will update with new expiration.
```
function getCookies(name) {
  const cookies = Cookies.get('alertAudio');
  if (cookies) {
    // Keep cookie fresh
    Cookies.set(name, cookies, attr);
  }
  return cookies;
}
```